README.md

```diff
@@ -80,6 +81,16 @@ empty_array_mt
 --------------
 **syntax:** `setmetatable({}, cjson.empty_array_mt)`
 
+**NOTE**: Use of this field is *discouraged* as of the `2.1.0.6` release. This
+field is considered *depcrecated*, and has been renamed to
+[array_mt](#array_mt) for expliciteness.
+
+[Back to TOC](#table-of-contents)
+
+array_mt
+--------
+**syntax:** `setmetatable({}, cjson.array_mt)`
+
 A metatable which can "tag" a table as a JSON Array in case it is empty (that is, if the
```

Comment:
@thibaultcha The user of cjson may not know if the tables are empty or not without testing for themselves. So requiring "empty" in the documentation is asking for such test on the user side.